cmd/fixtures_test/check_utils.go

```diff
@@ -337,6 +337,18 @@ func RunSingleFixtureTest(file string, t *testing.T) {
 		}
 		var fixtureSteps []FixtureStep
 		byteValue := ReadFile(file, t)
+
+		for idx, accountName := range accountNames {
+			tempName := "account" + strconv.Itoa(idx+1)
+
+			byteValue = []byte(strings.ReplaceAll(string(byteValue), string(tempName), string(accountName)))
```

Comment:
To be accurate, we need to replace the account key when using the value instead of replaceAll action in byte level.

Reply:
If there's a text that has same name with account key, it can cause an unexpected result.